packages/apps/marmalade-website/.eslintrc.json

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "next/core-web-vitals"
+}
```

Comment:
This should extend our @kadena-dev/eslint-config next config. I recommend referencing the kadena docs .eslintrc.json!

Reply:
yeah I planed to do it as a next step as in this pr some versions updates that are related to other packages was opening this initial pr with Next as it is